src/dos/program_mixer.cpp

```diff
@@ -881,15 +881,15 @@ void MIXER::AddMessages()
 	        "(must be between 0 and 100)");
 
 	MSG_Add("SHELL_CMD_MIXER_MISSING_GLOBAL_CROSSFEED_STRENGTH",
-	        "Missing global crossfeed strength after [color=white]X[reset] "
+	        "Missing global crossfeed strength after [color=white]x[reset] "
 	        "(must be between 0 and 100)");
 
 	MSG_Add("SHELL_CMD_MIXER_MISSING_GLOBAL_CHORUS_LEVEL",
-	        "Missing global chorus level after [color=white]C[reset] "
+	        "Missing global chorus level after [color=white]c[reset] "
```

Comment:
Hmm, I used uppercase so the singe-char command names stand out more in the error meesages.

I'm wondering if itwould be better to use uppercase everywhere instead?

Reply:
In my opinion it stands out well enough due to bright color. Uppercase is less convenient for the user, as he has to press SHIFT.

Reply:
Yeah, and our examples use all lowercase anyway. I was just wondering, but you're right 😄 

<img width="687" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/1b0718f4-db91-46c3-82f5-9e3323fafe32">

Lowercase it is then!

Reply:
I agree; easy on the eyes. 
(The height difference actually helps too, versus the height of the numbers).